### ISSUE TITLE
large datasets stop downloading at 1Gb

### ISSUE THREAD

**Issue by chrisbarnettster** [galaxycomputationalchemistry/usegalaxy-za#6]
Any downloads from server stop at around 1Gb. 
Tried saving individual datasets, exporting histories to file and happens in all cases. Save history to file looks as if it works until the file is uncompressed.

**Comment by chrisbarnettster** [comment]
this error found in Galaxy logs
uwsgi_response_write_body_do() TIMEOUT !!!
IOError: write error

**Comment by chrisbarnettster** [comment]
seem like a known issue, discussion on Gitter https://gitter.im/galaxyproject/admins?at=5e2b037cf196225bd64dae1b

should ensure X-Accel-Redirect configured in nginx

**Comment by chrisbarnettster** [comment]
discussed here in docs https://docs.galaxyproject.org/en/latest/admin/nginx.html
and examples in galaxy eu playbook
https://github.com/usegalaxy-eu/infrastructure-playbook/blob/9233fad69b39d04456e0f41a8b87eb12b2c524a2/templates/nginx/galaxy-main.j2
https://github.com/usegalaxy-eu/infrastructure-playbook/blob/ab9b39a448ab25ce7a158a4d6ca2782a2adbb515/group_vars/gxconfig.yml

**Comment by chrisbarnettster** [comment]
checked postgres backup and other config files

**Comment by chrisbarnettster** [comment]
in galaxy section of group_vars
`nginx_x_accel_redirect_base: /_x_accel_redirect `
and in templates/nginx/galaxy.j2
```
	location /_x_accel_redirect {
		internal;
		alias /;
		add_header X-Frame-Options SAMEORIGIN;
		add_header X-Content-Type-Options nosniff;
	}
```

**Comment by chrisbarnettster** [comment]
```
tmux
time ansible-playbook -i hosts playbook.yml --ask-vault-pass
```

**Comment by chrisbarnettster** [comment]
ERR_INVALID_RESPONSE when trying to download dataset

**Comment by chrisbarnettster** [comment]
update tempate as per https://docs.galaxyproject.org/en/latest/admin/nginx.html

**Comment by chrisbarnettster** [comment]
Add www-date to galaxy group - `sudo usermod -a -G galaxy www-data`

Now check perms on a dataset
`sudo -u www-data stat /cchem/galaxy/datasets/000/dataset_465.dat`

Restart galaxy
hit a spawn error. Stop all condas running with galaxy - sorry if those are job related.
Restart nginx `separately systemctl restart nginx`

5mins or so and still not working. 
I uploaded a dataset and then all data working and accessible.

Tried downloading large dataset. Slow but supports curl resume and wget resume.
